v2/dom.go

```diff
@@ -25,15 +25,13 @@
 // MDN Web Docs by Mozilla Contributors (https://developer.mozilla.org/en-US/docs/Web/API),
 // licensed under CC-BY-SA 2.5 (https://creativecommons.org/licenses/by-sa/2.5/).
 //
-//
-// Getting started
+// # Getting started
```

Comment:
These changes were done automatically by gofmt

Reply:
Applied them separately, in commit 3bbdb75377065c9b06d14b9de35fc5184701e5a1.